pkgs/applications/science/electronics/bitscope/packages.nix

```diff
@@ -0,0 +1,216 @@
+    } else if stdenv.system == "i686-linux" then fetchurl {
+      url = "http://bitscope.com/download/files/${name}_i386.deb";
+      sha256 = "0y9s7p36v7agqhi076711ds22nfxqmfgq13r01m57lpn9xfz5fja";
+    } else throw "no install instructions for ${stdenv.system}";
+  };
+
+  chart = let
+    name = "${toolName}_${version}";
+    toolName = "bitscope-chart";
+    version = "2.0.FK22M";
+  in mkBitscope {
+    inherit name toolName version;
+
+    meta = {
+      description = "Multi-channel waveform data acquisition and chart recording application";
+      homepage = "http://bitscope.com/software/chart/";
+    };
+
+    src = if stdenv.system == "x86_64-linux" then fetchurl {
+      url = "http://bitscope.com/download/files/${name}_amd64.deb";
+      sha256 = "08mc82pjamyyyhh15sagsv0sc7yx5v5n54bg60fpj7v41wdwrzxw";
+    } else if stdenv.system == "i686-linux" then fetchurl {
+      url = "http://bitscope.com/download/files/${name}_i386.deb";
+      sha256 = "08p6psh8g6pjp9gpdksb8y6nn9n4hv1j450v1nfhy25h2n8j0f03";
+    } else throw "no install instructions for ${stdenv.system}";
+  };
+
+  proto = mkBitscope rec {
+    # note: clicking on logo produces error
+    # TApplication.HandleException Executable not found: "http://bitscope.com/blog/DK/?p=DK15A"
+    name = "${toolName}_${version}";
+    toolName = "bitscope-proto";
+    version = "0.9.FG13B";
+
+    meta = {
+      description = "Prototype oscilloscope built using the BitScope Library";
+      homepage = "http://bitscope.com/blog/DK/?p=DK15A";
+    };
+
+    src = if stdenv.system == "x86_64-linux" then fetchurl {
+      url = "http://bitscope.com/download/files/${name}_amd64.deb";
+      sha256 = "1ybjfbh3narn29ll4nci4b7rnxy0hj3wdfm4v8c6pjr8pfvv9spy";
+    } else if stdenv.system == "i686-linux" then fetchurl {
+      url = "http://bitscope.com/download/files/${name}_i386.deb";
+      sha256 = "13v4r4x3h65ya5lxi5qgim62z7l7qjvyc53hzvkjzhzvd1ab5xzl";
+    } else throw "no install instructions for ${stdenv.system}";
+  };
+
+  console = let
+    # https://trello.com/c/sgH6tDdy/58-fb28a-bitscope-console
+    name = "${toolName}_${version}";
+    toolName = "bitscope-console";
+    version = "1.0.FK29A";
+  in mkBitscope {
+    inherit name toolName version;
+
+    meta.description = "Communications program designed to make it easy to talk to any model BitScope";
+    meta.longDescription = "BitScope Console is a communications program designed to make it easy to talk to any model BitScope via any supported communications link (serial, USB, LAN, Internet etc). Use it to interrogate the BitScope Virtual Machine to help write your own software for BitScope. It uses BitScope Link Library to facilitate communication. Note: you don't need the program to talk to most BitScopes, a simple terminal program will do but Console makes connection easier, uses the probe file specification and supports downstream communications to connected port adapters.";
```

Comment:
If you use double single-quotes, you can create a multi-line string of this longDescription that wraps at 80 chars.